src/constants/validation-constants.js

```diff
@@ -0,0 +1,31 @@
+import { Schema } from '../validation';
+
+const EMAIL_PROPERTY = 'email';
+const PASSWORD_PROPERTY = 'password';
+const REQUIRED_PASSWORD_LENGTH = 8;
+const CONFIRMED_PASSWORD_PROPERTY = 'confirmedPassword';
+
+export const LOGIN_CONSTANTS = {
+  EMAIL_PROPERTY,
+  PASSWORD_PROPERTY,
+  EMAIL_SCHEMA: new Schema().isRequired(),
+  PASSWORD_SCHEMA: new Schema().isRequired()
+};
+
+export const REGISTER_CONSTANTS = {
+  EMAIL_PROPERTY,
+  PASSWORD_PROPERTY,
+  REQUIRED_PASSWORD_LENGTH,
+  CONFIRMED_PASSWORD_PROPERTY,
+
```

Comment:
nitpicky but looks like extra white space

Reply:
Yeah, I can get rid of that, but it looked cluttered with all the all caps constants.